tree/ntuple/inc/ROOT/RNTupleProcessor.hxx

```diff
@@ -116,6 +242,19 @@ protected:
    ROOT::NTupleSize_t fCurrentEntryNumber = 0; //< Current processor entry number
    std::size_t fCurrentProcessorNumber = 0;    //< Number of the currently open inner processor
 
+   /////////////////////////////////////////////////////////////////////////////
+   /// \brief Check that the model provided for constructing the processor is bare.
+   ///
+   /// The use of bare models in the RNTupleProcessor is enforced because it adds additional field value pointer
+   /// management through RNTupleProcessorValuePtr. The use of pointers returned from a model's default entry (e.g.,
+   /// through RNTupleModel::AddField) may contain incorrect information, specifically when the corresponding field is
+   /// missing from an entry.
+   static void EnsureModelIsBare(const RNTupleModel &model)
```

Comment:
If I'm not missing something, this is probably better off as a static function in the cxx file directly.